src/pages/spawn/spawnHost/SpawnHostModal.tsx

```diff
@@ -119,7 +124,21 @@ export const SpawnHostModal: React.VFC<SpawnHostModalProps> = ({
     },
     refetchQueries: ["MyHosts", "MyVolumes", "GetMyPublicKeys"],
   });
-
+  const [migrateVolumeMutation, { loading: loadingMigration }] = useMutation<
+    MigrateVolumeMutation,
+    MigrateVolumeMutationVariables
+  >(MIGRATE_VOLUME, {
+    onCompleted() {
+      dispatchToast.success("Migrated volume onto host.");
```

Comment:
A successful response from the GraphQL mutation only indicates that the migration job was added to the queue. Something like this would be more accurate/helpful:

"Volume migration has been scheduled. A new host will be spawned and accessible on your Hosts page"